yatl_gen/pubspec.yaml

```diff
@@ -15,15 +15,15 @@ dependencies:
 
   glob: ^2.1.0
 
-  intl: ^0.19.0
+  intl: ^0.20.2
 
   recase: ^4.0.0
 
   xml: ^6.1.0
 
   yatl:
     git:
-      url: https://github.com/HrX03/yatl
+      url: https://github.com/AppleInternal/yatl.git
```

Comment:
Why do you change the target repo for this? If you have any local change in your repo it would be better to submit that too